bittide-shake/bin/Shake.hs

```diff
@@ -291,6 +291,7 @@ main = do
             checkpointsDir = synthesisDir </> "checkpoints"
             netlistDir = synthesisDir </> "netlist"
             reportDir = synthesisDir </> "reports"
+            ilaDir = synthesisDir </> "ila-data"
```

Comment:
I do think you improved the name, so you can remove `ilaDataPath` at line 329

Reply:
woops, didn't see that one?